src/Cake.NuGet/NuGetPackageInstaller.cs

```diff
@@ -56,38 +56,13 @@ public sealed class NuGetPackageInstaller : INuGetPackageInstaller
             ICakeLog log,
             ICakeConfiguration config)
         {
-            if (fileSystem == null)
-            {
-                throw new ArgumentNullException(nameof(fileSystem));
-            }
-            if (environment == null)
-            {
-                throw new ArgumentNullException(nameof(environment));
-            }
-            if (processRunner == null)
-            {
-                throw new ArgumentNullException(nameof(processRunner));
-            }
-            if (toolResolver == null)
-            {
-                throw new ArgumentNullException(nameof(toolResolver));
-            }
-            if (contentResolver == null)
-            {
-                throw new ArgumentNullException(nameof(contentResolver));
-            }
-            if (log == null)
-            {
-                throw new ArgumentNullException(nameof(log));
-            }
-
-            _fileSystem = fileSystem;
-            _environment = environment;
-            _processRunner = processRunner;
-            _toolResolver = toolResolver;
-            _contentResolver = contentResolver;
-            _log = log;
-            _config = config;
+            _fileSystem = fileSystem ?? throw new ArgumentNullException(nameof(fileSystem));
+            _environment = environment ?? throw new ArgumentNullException(nameof(environment));
+            _processRunner = processRunner ?? throw new ArgumentNullException(nameof(processRunner));
+            _toolResolver = toolResolver ?? throw new ArgumentNullException(nameof(toolResolver));
+            _contentResolver = contentResolver ?? throw new ArgumentNullException(nameof(contentResolver));
+            _log = log ?? throw new ArgumentNullException(nameof(log));
+            _config = config ?? throw new ArgumentNullException(nameof(config));
```

Comment:
This wasn't tested for null before. Is it correct that it now throws when it's null?